### PR TITLE
Second pass updating vendor prefixes

### PR DESCRIPTION
**Pull request by ntwb**
##### What issue # is this related to?

See #171, replaces PR #177
##### What is your WordPress.org username?

`netweb`
##### Questions:

Please make sure all the below checkboxes have been checked before submitting your PR.
- [x] Are all the above sections filled out appropriately?
- [x] Have you read [CONTRIBUTING.md](https://github.com/WordPress/twentyseventeen/blob/master/CONTRIBUTING.md)?
- [x] Did you maintain a new line at the end of each file modified/added?
- [x] If this adds new js dependencies, did you add them to [.jscsrc](https://github.com/WordPress/twentyseventeen/blob/master/.jscsrc) & [.jshintignore](https://github.com/WordPress/twentyseventeen/blob/master/.jshintignore)?

Please understand that because of the size of this project, your PR may go stale and before someone can review it. If this is the case please rebase/refresh your PR and resolve any conflicts.

Thank you for taking the time to contribute to Twenty Seventeen!

### CODE REVIEW

**Comment by ntwb** [comment]
Note: The vendor prefixes will still require further updates before the core merge

(I've a Grunt task "piggy backing" on WordPress Core's Autoprefixer to make this quick and easy)

File: style.css [hunk 1]
```diff
@@ -989,6 +1002,7 @@ a:hover .nav-title,
 
 .navigation-top .wrap {
 	padding: 0;
+	max-width: -webkit-calc( 1000px + 2.5em);
```
Comment (josephfusco) [inline style.css:1005]: While this PR is just related to prefixing, seems like we have some preexisting CSS guideline issues which are also being added to the prefixed values.

Extra space after opening parenthesis.

``` css
max-width: -webkit-calc( 1000px + 2.5em);
```

should be 

``` css
max-width: -webkit-calc(1000px + 2.5em);
```

**Comment by karmatosed** [comment]
-webkit-transition isn't needed anymore as far as I know.

**Comment by ntwb** [comment]
I'll refresh this pull request once #459 is merged, which in turn will address the issue @josephfusco noted above in https://github.com/WordPress/twentyseventeen/pull/379#pullrequestreview-4485633.

**Comment by davidakennedy** [comment]
@ntwb #459 merged! 😄

**Comment by ntwb** [comment]
> -webkit-transition isn't needed anymore as far as I know.

For newer Android & Chrome versions this is correct, as WP Core supports [Android >= 2.1](https://core.trac.wordpress.org/browser/trunk/Gruntfile.js#L29) and [Chrome >=21](https://core.trac.wordpress.org/browser/trunk/Gruntfile.js#L30) it is still included.

**Comment by ntwb** [comment]
Forgot the ping @karmatosed (see previous comment)

**Comment by ntwb** [comment]
Closing this in favour of _merge conflict free_ #475

**Pull request closed without merging**